### PR TITLE
[SettingsWindowController] Edit numberformatter to group by 4 digits …

### PR DESCRIPTION
**Pull request by bogren**
…and only allow numbers between 0-9999

### CODE REVIEW

**Comment by bogren** [comment]
Fixes #10 :v:

File: Luft/settings/SettingsWindowController.xib [hunk 1]
```diff
@@ -138,8 +147,12 @@
                             <constraint firstAttribute="width" constant="50" id="d8V-JU-5d5"/>
                             <constraint firstAttribute="height" constant="22" id="wxq-Wj-gQB"/>
                         </constraints>
+                        <animations/>
                         <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" editable="YES" sendsActionOnEndEditing="YES" state="on" borderStyle="bezel" alignment="right" drawsBackground="YES" id="fvk-7N-THA">
-                            <numberFormatter key="formatter" formatterBehavior="default10_4" numberStyle="decimal" minimumIntegerDigits="1" maximumIntegerDigits="2000000000" maximumFractionDigits="3" id="1hD-Dy-Apb"/>
+                            <numberFormatter key="formatter" formatterBehavior="custom10_4" localizesFormat="NO" allowsFloats="NO" groupingSize="4" minimumIntegerDigits="1" maximumIntegerDigits="2000000000" maximumFractionDigits="3" id="1hD-Dy-Apb">
+                                <real key="minimum" value="0.0"/>
+                                <real key="maximum" value="9999"/>
+                            </numberFormatter>
```
Comment (bogren) [inline Luft/settings/SettingsWindowController.xib:155]: :point_up:

File: Luft/settings/SettingsWindowController.xib [hunk 2]
```diff
@@ -163,8 +177,12 @@
                             <constraint firstAttribute="height" constant="22" id="9Ij-1s-XRl"/>
                             <constraint firstAttribute="width" constant="50" id="V1m-MS-pwi"/>
                         </constraints>
+                        <animations/>
                         <textFieldCell key="cell" scrollable="YES" lineBreakMode="clipping" selectable="YES" editable="YES" sendsActionOnEndEditing="YES" state="on" borderStyle="bezel" alignment="right" placeholderString="" drawsBackground="YES" id="3rM-38-ico">
-                            <numberFormatter key="formatter" formatterBehavior="default10_4" numberStyle="decimal" minimumIntegerDigits="1" maximumIntegerDigits="2000000000" maximumFractionDigits="3" id="ybN-MM-YJd"/>
+                            <numberFormatter key="formatter" formatterBehavior="custom10_4" localizesFormat="NO" allowsFloats="NO" groupingSize="4" minimumIntegerDigits="1" maximumIntegerDigits="2000000000" maximumFractionDigits="3" id="ybN-MM-YJd">
+                                <real key="minimum" value="0.0"/>
+                                <real key="maximum" value="9999"/>
+                            </numberFormatter>
```
Comment (bogren) [inline Luft/settings/SettingsWindowController.xib:185]: :point_up:

**Comment by k0nserv** [comment]
Awesome :)

**Pull request merged**